### PR TITLE
Clarify the impact on refreshes for node-stats telemetry

### PR DESCRIPTION
**Pull request by bartier**
This PR attempts to solve the issue #846.

The warning only emmits for clusters < version 7.2.0
The docs now contains a hint that this is only a problem for clusters < version 7.2.0

### CODE REVIEW

File: docs/telemetry.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ node-stats
 
 .. warning::
 
-    Using this telemetry device will skew your results because the node-stats API triggers additional refreshes.
+    With ``Elasticsearch < 7.2.0``, this telemetry device will skew your results because the node-stats API triggers additional refreshes.
```
Comment (hub-cap) [inline docs/telemetry.rst:80]: Lets not remove the word using, 
```
With ``Elasticsearch < 7.2.0``, using this ...
```

Comment (bartier) [reply docs/telemetry.rst]: Done =)

**Comment by bartier** [comment]
@hub-cap I updated the code. Can you check again? =)

**Comment by bartier** [comment]
@hub-cap Sorry, I've commited some unwanted changes, I'll fix.

**Comment by bartier** [comment]
@hub-cap Fixed unwanted changes. Can you check now?

**Comment by hub-cap** [comment]
Thank u very much for your contribution @bartier !

**Pull request merged**